ui/src/app/dashboard/dashboard.component.spec.ts

```diff
@@ -67,9 +69,23 @@ describe('DashboardComponent', () => {
     expect(testComponent).toBeTruthy();
   }));
 
+  it('should navigate to job-list page (fake) when status counts are clicked', fakeAsync(() => {
+    let countAnchor = fixture.debugElement.query(By.css("jm-dashboard a")).nativeElement;
+    countAnchor.click();
+    fixture.detectChanges();
+    tick();
+    expect(fixture.debugElement.query(By.css("div")).nativeElement.textContent).toEqual("fake job-list page");
```

Comment:
Could you make some assertions about the route, i.e. parsing the query parameter? Possibly by injecting the route into the `TestJobListComponent`, then getting a reference there? Alternatively, you cou[ld try to inject the ActiveRoute directly into the test case.

If you do this, you probably also want to select a more specific link on the page to click, e.g. in the "total summaries", find the link that says "failed", then click it.

If possible, another test case that clicks one of the aggregation cards would also be nice here.